### DISCUSSION TITLE
Apple Silicon build?

### DISCUSSION THREAD

**Discussion by esmorun** [michaelwillis/dragonfly-reverb#112] [Ideas]
Any chance we can have an Apple Silicon/M1 build of the plugins? :)

**Comment by falkTX** [comment]
Check the actions tab, that is already the case.

**Reply by falkTX** [reply]
last one being https://github.com/michaelwillis/dragonfly-reverb/suites/3697827552/artifacts/90207497

**Reply by esmorun** [reply]
Awesome, thanks! Maybe it should be mentioned in the readme so it's easier to find? :-)

**Reply by falkTX** [reply]
Up to the author, in my opinion a release ought to be made soon enough

**Reply by michaelwillis** [reply]
@esmorun Life has been busy and I haven't given this project much attention lately. The new Mac build was only recently introduced, so it is not documented yet. The whole readme needs an overhaul.

**Comment by michaelwillis** [comment]
The new [release 3.2.6](https://github.com/michaelwillis/dragonfly-reverb/releases/tag/3.2.6) now has a proper MacOS universal installer.
